src/functions/create-ms-team-notification.ts

```diff
@@ -3,10 +3,69 @@ export { createMsTeamNotification };
 async function createMsTeamNotification(
   event: any,
 ): Promise<any> {
-  console.log('event 👉', event);
+  //  let s:any='event';
+    //let obj:any = JSON.parse(s);
+  //  let myJSON:any = JSON.stringify(event);
+  //  let myObj:any = JSON.parse(myJSON);
+
+    //console.log(myJSON);
+
+    const s=event.Records[0];
```

Comment:
Format code by installing vscode prettier extension

And add this file(as in image), so that it auto formats on every save
![image](https://user-images.githubusercontent.com/24571343/179891510-4ecc3738-bd9f-4d1a-9d72-37564ff51bfd.png)